gui/gui-manager.cpp

```diff
@@ -299,10 +299,11 @@ void GuiManager::runLoop() {
 	}
 
 	Common::EventManager *eventMan = _system->getEventManager();
-	uint32 lastRedraw = 0;
-	const uint32 waitTime = 1000 / 60;
+	const uint32 targetFrameDuration = 1000 / 60;
```

Comment:
I’ve never really been clear on this, is this truly sufficiently accurate to not cause any rendering glitches even though the value is truncated?

Reply:
My take on it is that it does not matter:
* If wait for vsync is enabled, the delay duration does not matter as long as it does not cross a frame boundary, updateScreen is going to block until the right time anyways. The target frame duration value being a little too small helps not waiting for too long.
* If wait for vsync is disabled ... there's going to be tearing regardless.